Celeste.Mod.mm/Patches/Audio.cs

```diff
@@ -263,7 +263,9 @@ static class patch_Audio {
                 Audio.cachedEventDescriptions.Add(path, desc);
 
             } else if (status == RESULT.ERR_EVENT_NOTFOUND) {
-                Logger.Log(LogLevel.Warn, "Audio", $"Event not found: {path}");
+                if (path is not ("null" or "event:/none" or "event:/null")) {
```

Comment:
microlith said it was using event:/null somewhere in an unreleased map, I don't know if anyone else does though, might be worth removing. null is explicitly checked for in some places in vanilla (for example, using it as room music already doesn't show an error message because vanilla handles that), so it should be included here for consistency.

Reply:
oh gotcha. and yeah for `event:/null` it feels a bit odd to add a new convention for one unofficial usage. although I'm pretty sure `event:/none` was completely arbitrary too and never officially recommended lol